### PR TITLE
Upgrade Graal version

### PR DESCRIPTION
**Pull request by ChadKillingsworth**

### CODE REVIEW

**Comment by brad4d** [comment]
Hey @ChadKillingsworth 

Is this PR ready for review & merge? It looks like there are still some continuous integration failures.

Thanks for working on it.

**Comment by ChadKillingsworth** [comment]
No it's not even close. I'll change the title to say WIP, but I needed a PR to see the CI failures.

We can upgrade Graal pretty easily, but on windows we'll still be building with a very old SDK version. We have to upgrade to Java 11 to build on a newer windows SDK and that breaks the GWT builds.

**Comment by ChadKillingsworth** [comment]
@brad4d Ok now this is ready for review. This upgrades to the latest stable Graal version, but still builds on java 8 and the windows 7.1 SDK.

File: build-scripts/graal.js [hunk 1]
```diff
@@ -46,41 +46,44 @@ if (!fs.existsSync(TEMP_PATH)) {
...
+  '-H:+JNI',
+  '--no-server',
   '-H:+ReportUnsupportedElementsAtRuntime',
   '-H:IncludeResourceBundles=com.google.javascript.rhino.Messages',
   '-H:IncludeResourceBundles=org.kohsuke.args4j.Messages',
   '-H:IncludeResourceBundles=org.kohsuke.args4j.spi.Messages',
   '-H:IncludeResourceBundles=com.google.javascript.jscomp.parsing.ParserConfig',
   `-H:ReflectionConfigurationFiles=${path.resolve(__dirname, 'reflection-config.json')}`,
-  '-H:IncludeResources=(externs.zip)|(.*(js|txt))'.replace(/\|/g, () => {
+  '-H:IncludeResources=(externs.zip)|(.*(js|txt))'.replace(/[\|\(\)]/g, (match) => {
```
Comment (concavelenz) [inline build-scripts/graal.js:58]: Can you add a comment here as to what this code is attempting to do?

**Comment by concavelenz** [comment]
Left one comment, but otherwise looks good.

File: build-scripts/graal.js (hunk 1 above)
Comment (ChadKillingsworth) [reply build-scripts/graal.js:58]: The comment [just below this](https://github.com/google/closure-compiler-npm/pull/174/files/050e882a5409b9543754deea7248d1b3da8b5a36#diff-0a1cc5f9b2cbccfcc18b3fd7db39dc45L58) was designed to do that. Can you specify what's unclear from that?

It's all windows shell escaping requirements and truthfully it requires a lot of trial and error just to figure out how to make it work.

**Comment by lauraharker** [comment]
Tracked as internal Google issue http://b/159833201

File: build-scripts/graal.js (hunk 1 above)
Comment (concavelenz) [reply build-scripts/graal.js:58]: ok, good enough

**Pull request merged**